spec.bs

```diff
@@ -563,7 +564,9 @@ input {{SecurePaymentConfirmationRequest}} |data|, are:
 
 1. [=fetch an image resource|Fetch the image resource=] for the icon, passing «["{{ImageResource/src}}" →
     |data|["{{SecurePaymentConfirmationRequest/instrument}}"]["{{PaymentCredentialInstrument/icon}}"]]»
-    for *image*. If this fails, return `false`.
+    for *image*. If
+    |data|["{{SecurePaymentConfirmationRequest/instrument}}"]["{{PaymentCredentialInstrument/iconMustBeShown}}"]]
```

Comment:
I believe there is an extra closing bracket here that needs removed.

Reply:
Done